ppr-ui/src/components/mhrTransfers/DeathCertificate.vue

```diff
@@ -0,0 +1,180 @@
+              the death certificate matches the name displayed above exactly."
+              v-model="hasCertificate"
+              class="mt-0 pt-0 has-certificate-checkbox"
+              data-test-id="has-certificate-checkbox"
+            />
+          </v-col>
+        </v-row>
+      </v-form>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import { DatePicker } from '@bcrs-shared-components/date-picker'
+import { useInputRules } from '@/composables'
+import { computed, defineComponent, reactive, ref, toRefs, watch } from '@vue/composition-api'
+import { useActions } from 'vuex-composition-helpers'
+import { FormIF, MhrRegistrationHomeOwnerIF } from '@/interfaces' // eslint-disable-line no-unused-vars
+import { localTodayDate } from '@/utils'
+
+export default defineComponent({
+  name: 'DeathCertificate',
+  props: {
+    deceasedOwner: {
+      type: Object as () => MhrRegistrationHomeOwnerIF,
+      default: null
+    }
+  },
+  components: { DatePicker },
+  setup (props, context) {
+    const { customRules, required, maxLength } = useInputRules()
+
+    const {
+      setUnsavedChanges
+    } = useActions([
+      'setUnsavedChanges'
+    ])
+
+    const deathCertificateRef = ref(null)
+
+    const deathCertificateRules = computed(
+      (): Array<Function> => customRules(maxLength(20), required('Enter Death Certificate Number'))
+    )
+
+    const localState = reactive({
+      validateDeathCertificate: false, // NEW VALIDATOR REQUIRED
+      isFormValid: false, // Death Certificate form without Death Date Picker
+      isDeathCertificateFormValid: computed((): boolean => localState.isFormValid && !!localState.dateOfDeath),
+      deathCertificate: null,
+      dateOfDeath: null,
+      hasCertificate: false, // Will be used for validation on UI side only (original certificate checkbox)
+      deceasedOwner: props.deceasedOwner,
+      showFormError: computed(() => localState.validateDeathCertificate && !localState.isDeathCertificateFormValid),
+      maxDeathDate: computed((): Date => {
+        var dateOffset = 24 * 60 * 60 * 1000 // 1 day in milliseconds
+        var maxDate = new Date()
+        maxDate.setTime(maxDate.getTime() - dateOffset)
+        return maxDate
+      })
+    })
+
+    const hasError = (ref: any): boolean => {
+      return ref?.hasError
+    }
+
+    // Need function to validate
+
+    // Need function to clear data on undo/cancel
+
+    // Update deceased owner deathCertificateNumber when value changes
+    watch(
+      () => localState.deathCertificate,
+      (val: number) => {
+        localState.deceasedOwner.deathCertificateNumber = val
+        setUnsavedChanges(true)
+      }
+    )
+
+    // Update deceased owner deathDateTime when value changes
+    watch(
+      () => localState.dateOfDeath,
+      (val: string) => {
+        localState.deceasedOwner.deathDateTime = val
+        setUnsavedChanges(true)
+      }
+    )
```

Comment:
The watcher part works fine but the function is only mutating the components local copy of the prop you've passed down, if that makes sense. 
I think what you want to do is update the homeOwner (by id) not so much the local instance of the homeOwner. 
ie `watch( () => localState.dateOfDeath, (val: string) => { editSomeHomeOwner({ key: propertyName, val: someDate }) })`

Reply:
Yes, was going to update the local values as so, then once proper validations are in place, set the owner values outside the local instance when confirmed valid. So pretty much waiting on validations if you get what I mean

Reply:
Copy that 👍 
You can go ahead and set to store without validations, typically is allowed, we can implement validation after the fact.
 
Few reasons, we validate everything anywho, users can put in whatever they want typically.
We have draft filings which have zero validation, user can save whatever they want, to some extent. 

I know it seems odd but its more of the less the reason for the validations. 
We can always go two ways with it, enforce specific valid entries or allow the user to enter/store whatever but provide validations and information when things are invalid.

Reply:
Gotcha, on it